portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,10 +63,33 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("form-comment"); 
+    String name = request.getParameter("form-name"); 
```

Comment:
I just redirect without adding to the Datastore. May look into something else after I fix everything else (do you have any suggestions?)

Reply:
I would suggest to return an appropriate error response

Something like 

```
if (text == null || name == null) {
  response.sendError(<SOME_ERROR_CODE>);
  return;
}
```
https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html#sendError(int)
You can read more about error code in https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

Reply:
so right now if that happens (I changed the name property in html to test this), it'll just take you to a page with 
![image](https://user-images.githubusercontent.com/48307028/83649251-97d3f680-a57c-11ea-957c-812c570ae69c.png)
I used a 400 error because it would mean that the client had malformatted the name property. Please let me know if this doesn't make sense!